src/templates/admin/object_history.html

```diff
@@ -1,6 +1,6 @@
 {% extends "admin/base_site.html" %}
 {% load i18n admin_urls %}
-{% load url from future %}
+{#% load url from future %#}
```

Comment:
What does this mean?

Reply:
"future" is the registered tag which has some meaning but it not used in latest Django. So, I commented it.

Reply:
Would tell the meaning? Plus instead of commenting just delete it.